### DISCUSSION TITLE
Can the outputs be passed into a Powershell step?

### DISCUSSION THREAD

**Discussion by perryg-tiq** [tj-actions/changed-files#1117] [Q&A] [answered]
Hello,

Can an output such as '.any_changed ' be passed into a Powershell step?

- name: Powershell Test
        id: check_instance
        if: steps.changed-files.outputs.any_changed == 'true'
        shell: pwsh
        run: | ...

**Comment by jackton1** [comment, marked as answer]
@perryg-tiq Yes, the outputs can be passed into a Powershell step.
